### ISSUE TITLE
Paste from Apple Numbers not working correctly

### ISSUE THREAD

**Issue by romanstetsyk** [glideapps/glide-data-grid#783]
Pasting data from Apple Numbers doesn't seem to be working correctly. There is a new line character `\n` at the beginning and at the end of every value. For instance, a cell with the value of `'1'` becomes `'\n1\n'`. The values in the `onPaste` handler are readonly. Is there a way to transform data before pasting?

I used this event handler to console log the values that are being pasted:
```
onPaste={(target, values) => {
  console.log({ target, values });
  return true;
}}
```

Example of when I paste one column with numbers from Apple Numbers and Excel:
<img width="298" alt="image" src="https://github.com/glideapps/glide-data-grid/assets/25715951/51040c9b-6085-43a6-a6ac-970b68476527"> <img width="294" alt="image" src="https://github.com/glideapps/glide-data-grid/assets/25715951/c582e491-45b6-4b6f-8203-7aa8bb03b5f1">

https://github.com/glideapps/glide-data-grid/assets/25715951/16931320-d85e-46b4-ac02-192453328a8b

The link to [codesandbox](https://codesandbox.io/p/sandbox/glide-data-grid-modal-forked-lxxgmz?file=%2Fsrc%2FApp.tsx%3A96%2C11) to reproduce (obviously you'll need macOS and Apple Numbers to do that)

**Comment by jassmith** [comment]
Will fix, im working on this code anyway.

**Comment by jassmith** [comment]
Yeah it seems apple numbers is inserting random <p> tags for no reason other than to be extra fun. Will resolve.

**Comment by jassmith** [comment]
Fixed with ac85512
